### ISSUE TITLE
[high] : Many pages: Chrome + Jaws:  Back (<) control is not accessible using screen reader.

### ISSUE THREAD

**Issue by Stutikuls** [Expensify/App#8523]
## Action Performed:
1. Using Chrome + Jaws, open URL staging.new.expensify.com
2. Navigate to any chat window using Tab key and press Enter to select it.
3. Resize the browser.
4. Navigate to Back (<) button using Tab key and press Enter on it.
5. Observe the behavior on back control.

## Expected Result:
 Back (<) button should be accessible using screen reader.

## Actual Result:
Back (<) control is not accessible using screen reader.

## Workaround:
Can the user still use Expensify without this being fixed? Have you informed them of the workaround?

## Area issue was found in:
Where did we find this issue?

## Failed WCAG checkpoints
What WCAG checkpoints were failed?

## User impact:
Screen reader user may have difficulty going back if the back button is not accessible.

## Suggested resolution:
Ensure that the Back'< ' button can be accessible using Enter key when screen reader is enabled.                         
Refer to: https://www.w3.org/WAI/WCAG21/Techniques/general/G202

- Web
- Mobile Web

**Version Number:** v1.1.49-1
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**Issue reported by:** Stuti

https://user-images.githubusercontent.com/97600950/162134429-02b5d2c7-cf19-4173-943e-22540b65f3a0.mp4
